### PR TITLE
Team3 participation session

### PR DESCRIPTION
**Pull request by liux2**
This Pull Request adds the content needed for the participation section of the conduct guide.

### CODE REVIEW

File: conduct.md [hunk 1]
```diff
@@ -11,7 +11,39 @@
...
+Every team member needs to be actively and cooperatively involved with the
+projects at hand. This will ensure the flow of questions, ideas, and thoughts
+in a respectful manner. Our project at its root includes participation, candidness
+and insightful collaboration. Participating in project discussion means engaging
+in conversation, not just answering questions. The goal here is to show up and
+truly be all there.
+
+* **Acceptable Conduct (A):**
+  * *Members shall*
+    * Must contribute when designated to, in a timely manner
```
Comment (enpuyou) [inline conduct.md]: "shall must" seems awkward

File: conduct.md [hunk 2]
```diff
@@ -11,7 +11,39 @@
...
+    * Quick to respond in Slack to clarify confusing issues or assist other colleagues
```
Comment (enpuyou) [inline conduct.md]: Maybe rephrase to "respond quickly"

File: conduct.md [hunk 3]
```diff
@@ -11,7 +11,39 @@
...
+    * Everyone’s opinion must be considered
```
Comment (enpuyou) [inline conduct.md]: This line needs to be rephrased.

File: conduct.md [hunk 4]
```diff
@@ -11,7 +11,39 @@
...
+    * Willing to meetup and work outside of class
```
Comment (enpuyou) [inline conduct.md]: Be willing to

File: conduct.md [hunk 5]
```diff
@@ -11,7 +11,39 @@
...
+    * Approach issues and team members with professionalism
+    * Demonstrate continuous engagement with tasks
+    * Open minded
```
Comment (enpuyou) [inline conduct.md]: Be open-minded

File: conduct.md [hunk 6]
```diff
@@ -11,7 +11,39 @@
...
+  * *Members shall not*
+    * Demonstration of disrespect for team members
+    * Having an inappropriate professional boundary with team members
+    * Poor participation and attitude to their lab delegated tasks
+    * Patterns of poor communication or little effort for
+    collaboration in lab and Slack
+    * Disrupt working groups’ effectiveness by lack of collaboration
+    * Lack of insight into his or her level of competence
+    * Attempt to dominate a lab discussions
+    * No show in lab sessions without the consent of the professor
```
Comment (enpuyou) [inline conduct.md]: This section also should be rephrased as above. Each bullet point should connect to "Member shall not"

**Review by enpuyou — CHANGES_REQUESTED** [review]

File: conduct.md (hunk 1 above)
Comment (lussierc) [reply conduct.md]: I agree, I will change this @enpuyou.

**Review by ibarrezueta — APPROVED** [review]

**Review by enpuyou — APPROVED** [review]

**Review by corlettim — APPROVED** [review]

**Comment by gkapfham** [comment]
Hi @liux2 this branch is out of date with the base branch. Can you or a member of your team please update this? Thanks!

**Review by lussierc — APPROVED** [review]
The content overall looks pretty good and the formatting is good. I am approving this PR.
